src/taco-cli/cli/run.ts

```diff
@@ -186,7 +190,7 @@ class Run extends commands.TacoCommandBase {
             name: "local",
             run: Run.local,
             canHandleArgs(commandData: commands.ICommandData): boolean {
-                return !!commandData.options["local"];
+                return !!commandData.options["local"] || !!commandData.options["livereload"] || !!commandData.options["devicesync"];
```

Comment:
The intent is to support livereload/devicesync only in local scenarios.
I thought the indicator of a command to be ran remotely was only the presence of the 'remote' keyword.
Is this the appropriate method to use to check for that ? : 

```IPlatformWithLocation.location === PlatformHelper.BuildLocationType.Local```